docs/04_tying_in_the_market_maker.md

```diff
@@ -47,6 +48,7 @@ event LMSRMarketMakerCreation(
 );
 ```
 
+Rest of `migrations/07_create_lmsr_mm.js`:
```

Comment:
`/migrations/07_create_lmsr_mm.js`?

Reply:
Out of the blue a block of code appears without a hint of what file it is

Reply:
I mean, I think there is a "/" missing at the beginning :)

Reply:
Ah ok!

Reply:
Done